ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -491,8 +491,31 @@ StatusSectionLayout {
                 return chatContentModule.usersModule.model
             }
 
-            onAirdropClicked: communityTokensStore.airdrop(root.community.id,
-                                                           airdropTokens, addresses)
+            accountsModel: SortFilterProxyModel {
+                sourceModel: root.rootStore.accounts
+                proxyRoles: [
+                    ExpressionRole {
+                        name: "color"
```

Comment:
Somehow doesn't match with the base mock model

Reply:
I will unify it soon in upcoming pr. I opt for simple models that doesn't require using things like `Utils.getColorForId` in pure ui components.